setup.py

```diff
@@ -7,6 +7,15 @@
 from setuptools.dist import Distribution
 from typing import List
 
+
+
+has_mpi4py = False
```

Comment:
I don't like this logic. Ideally we should invert this. If netcdf-c has parallel support, we recommend users to install `mpi4py` and we do so in the tests. What do you think?

Reply:
I agree there's no need to check for mpi4py if netcdf-c doesn't have parallel support - so it is a bit backwards.

Reply:
Done.